<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelImage</class>
 <widget class="QMainWindow" name="ModelImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Image</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>441</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>testaudocad1_drawing.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DXF Drawing Preview</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
